Adds bounty page to docs site (#493)
onflow/flow: docs/content/bounties/index.mdx

---
title: Flow Bug Bounty Program

> description: Earn up to 100k by finding Flow bugs!

Reviewer: "Earn 100k or more by finding ...."

Reward for critical is minimum 100k

Reviewer: "Earn $100k USD value by finding Flow bugs" would be better too b/c can get rewards in different currencies.

> description: Earn up to 100k by finding Flow bugs!

Author: ```suggestion
description: Earn $100k USD by finding Flow bugs
```

> description: Earn up to 100k by finding Flow bugs!

Author: ```suggestion
description: Earn $100k USD by finding Flow bugs
```

sidebar_title: Bug Bounties
---

<div>

>   <Button feel="flat" size="large">

Reviewer 2: This button doesn't seem to do anything

Author: Not hooked up for security reasons. Will connect when approved.

> <div>
>   <Button feel="flat" size="large">
>     Submit a Bug
>   </Button>
> </div>

Author: ```suggestion
If you believe you may have found a security vulnerability in one of our products or platforms, send us an email: [security@dapperlabs.com](mailto:security@onflow.org?subject=Responsible%20disclosure)
```

<a href="/bounties/responsible-disclosure">
  Read our Responsible Disclosure Guidelines
</a>

## Rewards

<p>
  We run closed bug bounty programs, but beyond that, we also pay out rewards
  once per eligible bug to the first responsibly disclosing third party.

>   Rewards are based on the seriousness of the bug, but the minimum is $100 and

Reviewer: min = $2,500 USD value

>   we have and are willing to pay $100,000 or more at our sole discretion.

Reviewer: remove "we have" b/c i dont think we have yet, but we are willing to pay 100K+

## Rewards

<p>
  We run closed bug bounty programs, but beyond that, we also pay out rewards
  once per eligible bug to the first responsibly disclosing third party.

>   Rewards are based on the seriousness of the bug, but the minimum is $100 and

Author: ```suggestion
  Rewards are based on the seriousness of the bug, but the minimum is $2,500 USD and
```

>   we have and are willing to pay $100,000 or more at our sole discretion.

Author: ```suggestion
  we are willing to pay $100,000 or more at our sole discretion.
```
